search/docker-compose.yml

```diff
@@ -82,17 +82,21 @@ services:
   reactivesearch:
     image: my-reactivesearch
     container_name: reactivesearch
-    build: .
-    # volumes:
-    #   - ./:/code # for development
-    #   - /code/node_modules # an exception from the above volume
+    build:
+      context: .
+      dockerfile:  ${reactivesearch_dockerfile:-Dockerfile}
+    volumes:
+       - ./:/code # for development, unused in production
+       - /code/node_modules # an exception from the above volume
     ports:
       - "8080:80"
```

Comment:
The port should be different for dev and prod right? Like:
dev: "something:3000"
prod: "8080:80"

Reply:
Good point... it should expose the same port to nginx in either case, but we need to set the exposed port when using `yarn start`.

Reply:
This one it doesn't matter which port 3000 is mounted to, as nginx cannot do reverse proxy for a dynamic react server. We still have to use eg. localhost:1337 to access the reactivesearch instead of localhost/, which will return 502 Bad Gateway error

Reply:
> nginx cannot do reverse proxy for a dynamic react server

Why not?